### DISCUSSION TITLE
Feature Request: Add larger check button for daily check

### DISCUSSION THREAD

**Discussion by JayJJChen** [iSoron/uhabits#489] [Feature Requests]
I know I can add a widget to desktop to check a specific habit, but it's easy to misclick. Is it possible to have bigger check buttons in app, say in the bottom row, for easier daily check? As checking a habit in a specific day requires some precise tap&hold now.

**Comment by chennemann** [comment]
as far as I know we had this discussion already and dismissed the suggestion because the current implementation follows the material guidelines and should therefore not be changed. 

nevertheless could we think about implementing a kind of accessibility mode that increases the touch targets. I'm not sure whether this is useful or not since I personally don't have problems with clicking on the correct habit.

**Comment by iSoron** [comment]
@JayJJChen Thank you for the suggestion. Could you make a sketch (say, in MS Paint) of what your suggested bottom row could look like? Also, would making all the buttons larger, as @chennemann suggests, solve the issue?

@chennemann We dismissed the opposite suggestion -- making the checkmark buttons smaller. I would be fine with more accessible checkmark buttons. For example, I believe the checkmarks currently have fixed size, even if the user increases the system font. We could resize the buttons accordingly.

**Comment by chennemann** [comment]
I see. I just remembered some sort of discussion about this topic but couldn't recall what it was about exactly. 

In general it would be a good idea to have kind of a responsive design.

**Comment by JayJJChen** [comment]
hi @iSoron , thank you for your reply! I'm not familiar with material design. So the sketch maybe silly...
The idea is that maybe I can have 1 or 2 big buttons, I can only put them down in bottom right to be a shortcut, like picture below.
Or one icon(replace "meditate" with a + icon and remove the cook lunch on below image) to invoke more big buttons(like undone habits for the day/week), con is that it requires one more click, pro is that this button just invoke something the user hasn't finished, so it acts like a reminder, and reduces misclicks.

Also, when you click, maybe there will be a circle filling the button as visual feedback.

![loop](https://user-images.githubusercontent.com/34714360/55957755-ad5fd000-5c99-11e9-94a2-164f01f08500.png)

**Comment by chennemann** [comment]
thanks for the clarification. I personally don't think those buttons will provide any significant value. 

they even reduce the usability of the interface by covering parts of it. 

I think your point in having an easier way of checking/unchecking of habits is a valid one that should be addressed by increasing the touch target for easier accessibility. 

@iSoron will have the final word in this of course :smile:

**Comment by JayJJChen** [comment]
@chennemann thank you for you feedback. I'm not a designer nor an Android developer, I just like a button that's big and close to my thumb :P 
Your suggestion of the touch target solution could be a bigger checkbox before the habit, which will gray out if you reached your daily/weekly goal.

**Comment by chennemann** [comment]
fair enough :smile:

I was actually more thinking about increasing the height for each habit row and reduce the number of visible checkboxes. that would lead into a bigger touch target for each individual checkbox so it is easier to press the correct one. 

it won't solve the inconvenience of not being able to do everything with your thumb. on the other hand side I don't think we can find a better solution for that without compromising on other parts of the usability.
